src/app/work-item/work-item-list/work-item-list-entry.component.scss

```diff
@@ -0,0 +1,22 @@
+@import "../../../assets/stylesheets/custom";
+
+.type span{
```

Comment:
It's a SASS thing. In the original version "span" has a set of curly braces around it. In your version they were removed.